opencga-analysis/src/main/java/org/opencb/opencga/analysis/clinical/rga/RgaAnalysis.java

```diff
@@ -23,7 +21,7 @@ protected void check() throws Exception {
 
         this.rgaManager = new RgaManager(getCatalogManager(), getVariantStorageManager());
         study = getStudyFqn();
-        file = params.getString(RgaAnalysisParams.FILE);
+        file = params.getString(ParamConstants.FILE);
```

Comment:
Assuming this is an RGA parameter, shouldn't it be renamed to ParamConstants.RGA_FILE_PARAM?